config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/PercentStringArray
+SecureHeaders::Configuration.default do |config|
+  config.x_frame_options = "DENY"
+  config.x_content_type_options = "nosniff"
+  config.x_xss_protection = "1; mode=block"
+  config.x_download_options = "noopen"
+  config.x_permitted_cross_domain_policies = "none"
+  config.referrer_policy = %w[origin-when-cross-origin strict-origin-when-cross-origin]
+
+  google_analytics = %w[www.google-analytics.com ssl.google-analytics.com *.googletagmanager.com tagmanager.google.com *.googleusercontent.com *.gstatic.com]
+
+  config.csp = {
+    default_src: %w['none'],
```

Comment:
It's a shame that rubocop thinks that we shouldn't have strings that include `'` characters.

Reply:
Rubocop didn't pick anything up for me, have I missed something I should change, happy to restyle even without rubocop errors